api/apiCalls.js

```diff
@@ -0,0 +1,37 @@
+export function deleteData(path, id) {
+	return fetch(`${path}/${id}`, {
+		method: 'DELETE',
+	})
+}
+
+export function fetchData(path) {
+	return fetch(`/${path}`)
+}
+
+export function putData(path, item) {
+	return fetch(`/${path}/${item.id}`, {
+		method: 'PUT',
+		headers: {
+			'Content-Type': 'application/json',
+		},
+		body: JSON.stringify({
+			title: item.title,
+			price: item.price,
+			url: item.url,
+		}),
```

Comment:
Nitpick: but based on the implementation it looks like these functions are executing business logic and can't be reused. Consider naming them more explicitly in such case and then there is little point in asking for `path` to be passed into the function as it's only tailored for one specific endpoint. (the body construction)

Reply:
I could be creating or editing carousel photos with same functions. I might need to change some parameters though, because carousel photo has no price so it would be left out.

Reply:
My idea was that in a bit bigger application you'd be better of having a function, which just accepts `body` or `payload` parameter and is more reusable.
Something like:
```js
export function put(path, body) {
  return fetch(path, {
    method: 'PUT',
    headers: {
      'Content-Type': 'application/json',
    },
    body,
  });
}
```